projects/backend/capi/articleTypePicker.ts

```diff
@@ -34,6 +34,10 @@ const articleTypePicker = (article: IContent): ArticleType => {
     const isReview: boolean = isTagPresent('tone/reviews')
     const isRecipe: boolean = isTagPresent('tone/recipes')
     const isMatchResult: boolean = isTagPresent('tone/matchreports')
+    const isCorrection: boolean = isTagPresent('theguardian/series/correctionsandclarifications')
+    const isBirthday: boolean = isTagPresent('news/birthdays')
+    const isSoundAndVision: boolean = isTagPresent('tv-and-radio/series/the-10-best-tv-shows-in-the-uk-this-week')
```

Comment:
Just out of curiosity, do we not need to use these new `const` somewhere?

Reply:
As part of my work on the new no byline template they will be used to assign particular articles to a new article type.

Reply:
Got it, thank you @aoifemcl15